src/main/java/com/netflix/imflibrary/imp_validation/IMFMasterPackage.java

```diff
@@ -61,7 +52,7 @@ public IMFMasterPackage(List<ResourceByteRangeProvider> resourceByteRangeProvide
                 assetMaps.add(resourceByteRangeProvider);
             } else if (PackingList.isFileOfSupportedSchema(resourceByteRangeProvider)) {
                 packingListsResourceByteRangeProviders.add(resourceByteRangeProvider);
```

Comment:
Should we consider renaming 'packingListsResourceByteRangeProviders' to 'packingLists'?